### DISCUSSION TITLE
blog/why-i-use-vim-and-why-you-should-too/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#2] [Q&A]
# blog/why-i-use-vim-and-why-you-should-too/

VIM or Vi Improved, is a free and open-source text editor for the terminal written by Bram Moolenaar. It is a highly powerful and versatile text editor that has managed to gather a devoted following among many people. Known for its efficiency, speed, and extensive customization options, Vim offers a unique modal editing approach that distinguishes it from traditional text editors.
Although I was familiar with Vim since 2020, I never thought of actually using it.

http://localhost:1313/blog/why-i-use-vim-and-why-you-should-too/

**Comment by darkrai343** [comment]
This made me want to try Vim. Good read!

**Reply by Jitesh117** [reply]
Glad to hear!
